packages/itwinui-react/src/core/Carousel/Carousel.tsx

```diff
@@ -98,8 +98,9 @@ export const Carousel = Object.assign(
         return;
       }
 
-      if (event.key === 'ArrowLeft' || event.key === 'ArrowRight') {
-        setKeysPressed((old) => ({ ...old, [event.key]: true }));
+      const { key } = event;
```

Comment:
I think you could write this as:
```ts
const key = event.key;
```

We do this in more places than destructing event props.